examples/with-typescript/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "dev": "concurrently \"tsc --watch\" \"next\""
```

Comment:
True! It's my commit but on @giacomorebonato fork that I don't have push access to, can @giacomorebonato rebase it or do I have to do it and make a new PR to that fork? This is a bit outside my comfort zone with git.